config/raw_metadata_test.py

```diff
@@ -48,7 +49,7 @@ def test_validate_metadata(self):
                             msg="Can not open URL: <{}>. Check how you collect URLs".format(
                                 metadata[1]['url']))
 
-            html_source = requests.get(metadata[1]['url']).text
+            html_source = html.unescape(requests.get(metadata[1]['url'], cookies={'beget': 'begetok'}).text)
```

Comment:
@dmitry-uraev take a look.
1. separate PR with unescaping the content, extend the json with cookies, use cookies if specified in the json file - @asbabiy  
2. @dmitry-uraev to review and merge it

Reply:
addressed in the separate PR